common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -46,7 +46,7 @@
 	public enum TimeoutPolicy {RETRY, TIME_OUT_WF, ALERT_ONLY}
 
 	@ProtoEnum
-	public enum RetryLogic {FIXED, EXPONENTIAL_BACKOFF}
+	public enum RetryLogic {FIXED, EXPONENTIAL_BACKOFF, CUSTOM}
```

Comment:
Upon further deliberation, we realized that the retry strategy customization cannot be used to its full potential unless both the task owners and task users can avail of the customization. 
To that effect, it would be beneficial to add the RetryLogic to the `WorkflowTask` to enable the customization of the retry strategy at a workflow level on a given task. 
Further to enable the use of this customization, the RetryLogic enum can be extended to add `UNSPECIFIED` as a value which could be the default for the WorkflowTask and to maintain backward compatibility.

Reply:
Additionally, validation can also be added to the TaskDefinition such that a task def owner can only select the UNSPECIFIED retry logic if the retry delay seconds is not being overridden in their task definition.

Reply:
Added the relevant logic.